sourcerer-eventstore/src/main/java/org/elder/sourcerer/eventstore/EventStoreEventRepository.java

```diff
@@ -26,6 +26,8 @@
 import org.elder.sourcerer.EventRepository;
 import org.elder.sourcerer.EventSubscriptionUpdate;
 import org.elder.sourcerer.ExpectedVersion;
+import org.elder.sourcerer.ExpectedVersionType;
+import org.elder.sourcerer.exceptions.AtomicWriteException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.elder.sourcerer.exceptions.AtomicWriteException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.elder.org/coding_rules#rule_key=squid%3AUselessImportCheck)